contrib/internal/httptrace/config.go

```diff
@@ -27,7 +27,6 @@ const (
 )
 
 // defaultQueryStringRegexp is the regexp used for query string obfuscation if `envQueryStringRegexp` is empty.
-// The regexp is taken from https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2490990623/QueryString+-+Sensitive+Data+Obfuscation
```

Comment:
We could link to https://docs.datadoghq.com/tracing/configure_data_security#redacting-the-query-in-the-url instead.